pagopa/io-web-profile
# [SIW-1543]: remove IT Wallet references from dictionary

## Short description

The dictionary has been updated to remove references to `"IT Wallet"` for consistency with the app. Additionally, the layout on the `revoca-wallet` page has been removed to avoid displaying incorrect metadata in the browser tab.
Error 500 in wallet status api does not redirect to generic error page.

## List of changes proposed in this pull request

- Updated the `it.json` dictionary file to remove `"IT Wallet"` references.
- Removed the layout from the `revoca-wallet` page that was causing incorrect metadata to appear in the browser tab.

## How to test

- Follow the revoke flow in the web application to ensure that the `"IT Wallet"` text is no longer displayed.
- Check the browser metadata on the `revoca-wallet` page to confirm that it no longer shows `"C'è un problema temporaneo"`.
